src/tests/ftest/server/metadata.py

```diff
@@ -35,8 +36,11 @@ def run_ior_loop(manager, uuids, tmpdir_base):
         manager.job.dfs_cont.update(cont_uuid, "ior.cont_uuid")
 
         # Create a unique temporary directory for the the manager command
-        tmp_dir = mkdtemp(dir=tmpdir_base)
-        manager.tmpdir_base.update(tmp_dir, "tmpdir_base")
+        if hasattr(manager, "tmpdir_base"):
+            tmp_dir = mkdtemp(dir=tmpdir_base)
+            manager.tmpdir_base.update(tmp_dir, "tmpdir_base")
```

Comment:
Since the test uses clush now, the job manager won't have a `tmpdir_base` so I think we can remove all references to `tmpdir_base` in this file. @kccain Any reason I've overlooked that we still need this?

Reply:
Thanks for the review @daltonbohning. We could consider removing tmpdir_base. I left it here without thinking about it too much probably because to solve this problem I tried different managers, each with their own particular problems, some of which use tmpdir_base. e.g., had tried with Orterun (that the test used to begin with), Mpirun / openmpi, and Mpirun / mpich. Eventually I found that Clush worked.

I can push another version of the patch that removes tmpdir_base in this file.